ytmusicapi/mixins/browsing.py

```diff
@@ -341,11 +341,11 @@ def get_album_browse_id(self, audioPlaylistId: str) -> str:
         """
         params = {"list": audioPlaylistId}
         response = self._send_get_request(YTM_DOMAIN + "/playlist", params)
-        matches = re.search(r"\"MPRE.+?\"", response.text.encode("utf8").decode("unicode_escape"))
-        browse_id = None
-        if matches:
-            browse_id = matches.group().strip('"')
-        return browse_id
+
+        if (start := response.text.find('MPREb_')) == -1:
```

Comment:
I feel like this will break some edge cases and is too simple.

Can you elaborate on the warning you were getting, perhaps we can solve it a different way

Reply:
When running "coverage run -m unittest test.py" from the tests directory. 

```
(python312) birdwell@JCBMPro tests % coverage run -m unittest test.py
s..../Users/birdwell/PycharmProjects/ytmusicapi-contrib/ytmusicapi/ytmusicapi/mixins/browsing.py:345: DeprecationWarning: invalid escape sequence '\/'
  matches = re.search(r"\"MPRE.+?\"", response.text.encode("utf8").decode("unicode_escape"))
....
```

I can't seem to reproduce it outside of coverage/unittest

Reply:
Yes, I see it here on CI as well. It's pretty strange, because the same regex works in a simple isolated script without issue

Reply:
your proposed approach does not work with `OLAK5uy_nbMYyrfeg5ZgknoOsOGBL268hGxtcbnDM` from #470 and would thus introduce a regression

Reply:
If you don't mind I would keep the warning for now and focus on the main goal of the PR